doc/admin-guide/files/sni.yaml.en.rst

```diff
@@ -98,11 +98,11 @@ client_key                The file containing the client private key that corres
                           :ts:cv:`proxy.config.ssl.client.private_key.filename` is used.
 
 
-disable_h2                :code:`true` or :code:`false`.
+http2                     :code:`on` or :code:`off`.
```

Comment:
Prob should note the default here.

Reply:
Except, there is no default :) The default varies with whatever the ports specification says. This is really an override. But I can try to add something to explain that.